docker-compose/database/3_add_statistics_procedure.sql

```diff
@@ -0,0 +1,90 @@
+-- Shanoir NG - Import, manage and share neuroimaging data
+-- Copyright (C) 2009-2019 Inria - https://www.inria.fr/
+-- Contact us on https://project.inria.fr/shanoir/
+-- 
+-- This program is free software: you can redistribute it and/or modify
+-- it under the terms of the GNU General Public License as published by
+-- the Free Software Foundation, either version 3 of the License, or
+-- (at your option) any later version.
+-- 
+-- You should have received a copy of the GNU General Public License
+-- along with this program. If not, see https://www.gnu.org/licenses/gpl-3.0.html
+
+use datasets;
+
+delimiter //
+
```

Comment:
Have you tested to enter SQL injection like, or comments in the entries to check security ?

Reply:
J'ai checké, 
```StoredProcedureQuery query [...]
query.setParameter(1, studyNameInRegExp);```
Permet de se prémunir des injections SQL.